src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.scss

```diff
@@ -2,37 +2,33 @@
 
 .trackConfig {
   display: grid;
-  grid-template-columns: 146px 343px 1fr;
+  grid-template-columns: 140px 250px 1fr;
   background: #f1f2f4;
   box-shadow: 1px 1px 10px #6f8190;
   position: absolute;
-  left: -614px;
+  left: -395px;
```

Comment:
`left` with the value equal to the width of the panel in pixels was a weird way to define the position. Try `right` to just define the width of the offset, with no concern about the width of the config panel itself.